auction_keeper/gas.py

```diff
@@ -81,7 +84,7 @@ def __str__(self):
         elif self.fixed_gas:
             retval = f"Fixed gas price {round(self.fixed_gas / self.GWEI, 1)} Gwei "
         else:
-            retval = f"Default gas 10 Gwei "
+            retval = f"Node gas price "
```

Comment:
Consider writing the gas price value in the output for greater awareness. 
i.e. `retval = f"Node gas price {initial_price} Gwei"`

Reply:
The node price is constantly changing; logging the amount upon startup seems misleading.

Reply:
Still, it'd be nice to have some feedback to ensure their node supports the JSON-RPC call.  I'll put some thought into this.